conf/application.conf

```diff
@@ -27,6 +27,12 @@ play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.FrontendFilters"
 
 play.application.loader = "uk.gov.hmrc.play.bootstrap.ApplicationLoader"
 
+play.filters.enabled += "uk.gov.hmrc.play.bootstrap.frontend.filters.AllowlistFilter"
+play.filters.enabled += "uk.gov.hmrc.play.bootstrap.frontend.filters.SessionIdFilter"
+bootstrap.filters.allowlist.ips = "127.0.0.1"
```

Comment:
it will never actually use the list of IPs here either apart from in Prod where we will override so worth just making this empty string